Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -739,6 +799,43 @@ void WiiSockMan::Convert(WiiSockAddrIn const& from, sockaddr_in& to)
   to.sin_port = from.port;
 }
 
+s32 WiiSockMan::ConvertEvents(s32 events, ConvertDirection dir)
+{
+  struct
+  {
+    int native;
+    int wii;
+  } mapping[] = {
```

Comment:
I wonder if this could (or even should) be `static` (and maybe `const`), perhaps even `constexpr`?

Reply:
`constexpr` seems to work